vars/atomicHostTests.groovy

```diff
@@ -0,0 +1,107 @@
+import org.centos.Utils
+import org.centos.pipeline.PipelineUtils
+
+def call(body) {
+
+    def config = [:]
+    body.resolveStrategy = Closure.DELEGATE_FIRST
+    body.delegate = config
+    body()
+
+    def getUtils = new Utils()
+    def getPipelineUtils = new PipelineUtils()
+    def getMessage = new Messaging()
+    def current_stage = 'atomic-host-tests'
+
+    try {
+        stage(current_stage) {
+            // Change display
+            currentBuild.displayName = "Build#: ${env.BUILD_NUMBER} - Branch: ${env.branch} - Package: ${env.fed_repo}"
+            currentBuild.description = "Stage: ${current_stage}"
+            // Set groovy and env vars
+            env.task = "./ci-pipeline/tasks/atomic-host-tests"
+            env.playbook = "ci-pipeline/playbooks/system-setup.yml"
+
+            // Send integration test running message on fedmsg
+            env.topic = "${MAIN_TOPIC}.ci.pipeline.compose.test.integration.running"
+            messageProperties = "topic=${topic}\n" +
+                    "build_url=${BUILD_URL}\n" +
+                    "build_id=${BUILD_ID}\n" +
+                    "compose_url=http://artifacts.ci.centos.org/artifacts/fedora-atomic/${branch}/ostree\n" +
+                    "compose_rev=${commit}\n" +
+                    "branch=${branch}\n" +
+                    "ref=fedora/${branch}/${basearch}/atomic-host\n" +
+                    "rev=${fed_rev}\n" +
+                    "repo=${fed_repo}\n" +
+                    "namespace=${fed_namespace}\n" +
+                    "username=fedora-atomic\n" +
+                    "test_guidance=''\n" +
+                    "status=${currentBuild.currentResult}"
+            messageContent = ''
+            getMessage.sendMessage([topic:"${env.topic}",
+                                    provider:"${env.MSG_PROVIDER}",
+                                    msgType:'custom',
+                                    msgProps:messageProperties,
+                                    msgContent:messageContent])
+
+            // Provision resources
+            env.DUFFY_OP = "--allocate"
+            getUtils.duffyCciskel([stage:current_stage, duffyKey:'duffy-key', duffyOps:env.DUFFY_OP])
+
+            echo "Duffy Allocate ran for stage ${current_stage} with option ${env.DUFFY_OP}\r\n" +
+                    "ORIGIN_WORKSPACE=${env.ORIGIN_WORKSPACE}\r\n" +
+                    "ORIGIN_BUILD_TAG=${env.ORIGIN_BUILD_TAG}\r\n" +
+                    "ORIGIN_CLASS=${env.ORIGIN_CLASS}"
+
+            def job_props  = "${env.ORIGIN_WORKSPACE}/job.props"
+            def job_props_groovy = getUtils.convertProps(job_props)
+            load(job_props_groovy)
+
```

Comment:
We could always break down stuff more I think this is good enough for now given the complete difference in each stage of what a message contains.  I can make the display different of course this doesn't work very well without env vars or ${config<var>} name all over the place